store/src/db.rs

```diff
@@ -27,13 +34,42 @@ impl Db {
 
         info!(sqlite = format!("{}", config.sqlite.display()), "Connected to the DB");
 
+        // Feature used to support `IN` and `NOT IN` queries
+        conn.interact(|conn| array::load_module(conn))
+            .await
+            .map_err(|_| anyhow!("Loading carray module failed"))??;
```

Comment:
An alternative implementation would use the `post_hook`. I have to look at that later on, but for now this seems to be working.